packages/keycloak-user-management/src/components/Credentials/index.tsx

```diff
@@ -56,13 +57,9 @@ export const submitForm = (
   values: UserCredentialsFormFields,
   props: CredentialsPropsTypes
 ): void => {
-  const { serviceClass, match, accessToken } = props;
+  const { serviceClass, match, accessToken, keycloakBaseURL } = props;
   const userId = match.params.userId;
-  const serve = new serviceClass(
-    accessToken,
-    `/users/${userId}/reset-password`,
-    'https://keycloak-stage.smartregister.org/auth/admin/realms/opensrp-web-stage'
-  );
+  const serve = new serviceClass(accessToken, `/users/${userId}/reset-password`, keycloakBaseURL);
```

Comment:
Can we please not have this magic string here?  All URLs should be declared as constants.